### PR TITLE
fix highlight color for ALEWarningSign, also set it for ALEErrorSign, ALEInfoSign

### PR DESCRIPTION
**Pull request by ArtAndreev**

### CODE REVIEW

**Comment by ArtAndreev** [comment]
Also I set another sign colors like in corresponding messages.

File: colors/onedark.vim [hunk 1]
```diff
@@ -577,6 +577,9 @@ call s:h("GitGutterDelete", { "fg": s:red })
 call s:h("ALEError", { "fg": s:red, "gui": "underline", "cterm": "underline" })
 call s:h("ALEWarning", { "fg": s:yellow, "gui": "underline", "cterm": "underline" })
 call s:h("ALEInfo", { "gui": "underline", "cterm": "underline" })
+call s:h("ALEErrorSign", { "fg": s:red })
+call s:h("ALEWarningSign", { "fg": s:yellow })
+call s:h("ALEInfoSign", { })
```
Comment (joshdick) [inline colors/onedark.vim:582]: Thanks for opening this PR!

I'm curious whether this line with no colors specified has any actual effect, or if you just included it for completeness?

Comment (ArtAndreev) [reply colors/onedark.vim:582]: Yes, I added it for completeness.

Variables for line and sign differ in underlining, all other properties are the same. In this case, `fg`.

With this line info sign has the following highlighting:
```
:hi ALEInfoSign
ALEInfoSign    xxx cleared
```

It looks valid.

**Comment by joshdick** [comment]
Thanks for the report and the PR!

**Pull request merged**